back/config/schedule.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Use this file to easily define all of your cron jobs.
+# Learn more: http://github.com/javan/whenever
+
+every 6.hours do
```

Comment:
Just a minor point. Wonder if making this configurable through .env would be an idea? Just in case we need to experiment with timings.

Reply:
Actually, I just realized that the automatic setup of cron tasks cannot work that way. So I'll leave things as they are for now. In the worst case, we'll just do it as we always did, by manually editing the crontab on the first node of each cluster.

Reply:
We also have regular unix cron jobs running on every single cluster with certain rake tasks. should we maybe unify our approach to not have both systems running? They seem to be do the very same thing as far as I can see.

Reply:
Yes, this is what I was referring to by "manually modify the crontab on the first node of each cluster". It was an attempt to make things more automatic/reproducible, but that's more work than I expected. So I'll probably give up... for now. 

But curious, how did you end here? 😁🧐

Reply:
Solely by accident and curiosity :)